.github/workflows/publish_connectors.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
           github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
           metadata_service_gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
+          sentry_dsn: "https://f2e9aeb6b24b4024b09679201c7f563e@o1009025.ingest.sentry.io/4505596642983936"
```

Comment:
Setting this in this workflow is what make the reporting only enable for publish right?

Reply:
Yep!